draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1006,10 +1023,10 @@ cookies" (either a registrable domain, or the empty string):
     1.  Let `origin` be the origin of `item`'s URI if `item`'s sandboxed origin
         browsing context flag is set, and `item`'s origin otherwise.
 
-    2.  If `origin`'s host's registrable domain is not an exact match for
-        `top-origin`'s host's registrable domain, return the empty string.
+    2.  If `origin` is not same-site with `top-origin`, return a globally
```

Comment:
RFC6454 says "generate a fresh globally unique identifier and return that value." I'd suggest using that so it's clear that this identifier should be different from others.

Reply:
That's a good point.